libs/ui-watt/src/lib/components/tabs/tab/tab.component.ts

```diff
@@ -24,4 +31,9 @@ export class WattTabComponent {
   @Input() label = '';
   @ViewChild('templateRef') public templateRef: TemplateRef<unknown> | null =
     null;
+  @Output() changed = new EventEmitter();
```

Comment:
Suggestion: add type for the event emitter
```suggestion
  @Output() changed = new EventEmitter<void>();
```